eng/pipelines/common/global-build-job.yml

```diff
@@ -36,5 +36,14 @@ jobs:
     - template: /eng/pipelines/common/clone-checkout-bundle-step.yml
 
     # Build
-    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -c ${{ parameters.buildConfig }} -runtimeConfiguration ${{ parameters.runtimeConfig }} -arch ${{ parameters.archType }}
+    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -c ${{ parameters.buildConfig }} -runtimeConfiguration ${{ parameters.runtimeConfig }} -arch ${{ parameters.archType }} -bl
```

Comment:
I would instead recommend using the -ci flag which does extra logging, clean-up, makes sure the dotnet we acquired is on the path and isolate the nuget cache, plus, it generates binlogs.